FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/AzureTableClientFactory.cs

```diff
@@ -16,19 +16,19 @@ public class AzureTableClientFactory : IAzureTableClientFactory
     {
         private const string StorageEmulatorConnectionString = "UseDevelopmentStorage=true";
         private readonly ITokenCredentialProvider _credentialProvider;
-
         private readonly string _tableUrl;
+        private readonly string _internalConnectionString;
```

Comment:
move the `StorageEmulatorConnectionString` const string to a common file and use it?

Reply:
Thanks, moved it to common file.